src/renderer/src/pages/ChatPage/components/InputForm/TextArea.tsx

```diff
@@ -313,10 +408,15 @@ export const TextArea: React.FC<TextAreaProps> = ({
             onDrop={handleDrop}
             required
             rows={3}
+            style={{ height: `${textareaHeight}px` }}
           />
+        </div>
 
-          {/* Model Selector, Thinking Mode, and Plan/Act Toggle at the bottom of textarea */}
-          <div className="absolute left-4 bottom-3.5 flex items-center gap-2.5 z-10 pointer-events-auto">
+        {/* Controls at the bottom */}
+        <div
+          className={`absolute bottom-0 left-0 right-0 flex items-center justify-between px-4 py-2 bg-gray-50 dark:bg-gray-800 rounded-b-lg ${isScrolledToBottom ? '' : 'border-t border-gray-200 dark:border-gray-700'}`}
```

Comment:
Why not eliminate borders and keep it simple?

```suggestion
          className={`absolute bottom-0 left-0 right-0 flex items-center justify-between px-4 py-2 bg-gray-50 dark:bg-gray-800 rounded-b-lg`}
```

Reply:
I initially included the borders from a usability perspective to visually indicate to users which areas aren't part of the editable textarea. However, I do appreciate your point about maintaining simplicity, so I'm fine with removing the borders if that's preferred.

Reply:
For reference, I modeled this implementation after Slack's chat UI approach. In the latest commit, I noticed the boundary line wasn't dynamically displaying during line breaks, so I added a small fix for that issue. As mentioned in my previous comment, I'm fully on board with the simplification approach by removing borders - if you prefer a borderless design, I'm happy to implement that change.